stories/icon-tab-bar/icon-tab-bar.stories.js

```diff
@@ -0,0 +1,1434 @@
+                <span class="fd-icon-tab-bar__badge"></span>
+            </a>
+        </li>
+    </ul> 
+</div>
+<br><br>
+<div class="fd-icon-tab-bar fd-icon-tab-bar--icon-only">
+  <ul role="tablist" class="fd-icon-tab-bar__header">
+    <li role="presentation" class="fd-icon-tab-bar__item fd-icon-tab-bar__item--positive">
+      <a role="tab" class="fd-icon-tab-bar__tab" href="#" id="tab12">
+        <div class="fd-icon-tab-bar__container">
+          <span class="fd-icon-tab-bar__icon">
+                        <i class="sap-icon--history" role="presentation"></i>
+                    </span>
+          <span class="fd-icon-tab-bar__badge fd-icon-tab-bar__badge--animated"></span>
+          <span class="fd-icon-tab-bar__counter">2</span>
+        </div>
+      </a>
+    </li>
+    <li role="presentation" class="fd-icon-tab-bar__item fd-icon-tab-bar__item--negative">
+      <a role="tab" class="fd-icon-tab-bar__tab" href="#" aria-selected="true" id="tab13">
+        <div class="fd-icon-tab-bar__container">
+          <span class="fd-icon-tab-bar__icon">
+                        <i class="sap-icon--card" role="presentation"></i>
+                    </span>
+          <span class="fd-icon-tab-bar__badge fd-icon-tab-bar__badge--animated"></span>
+          <span class="fd-icon-tab-bar__counter">23</span>
+        </div>
+      </a>
+    </li>
+    <li role="presentation" class="fd-icon-tab-bar__item fd-icon-tab-bar__item--critical">
+      <a role="tab" class="fd-icon-tab-bar__tab" href="#" id="tab14">
+        <div class="fd-icon-tab-bar__container">
+          <span class="fd-icon-tab-bar__icon">
+                <i class="sap-icon--cart" role="presentation"></i>
+          </span>
+          <span class="fd-icon-tab-bar__badge fd-icon-tab-bar__badge--animated"></span>
+          <span class="fd-icon-tab-bar__counter">10</span>
+        </div>
+      </a>
+    </li>
+    <li role="presentation" class="fd-icon-tab-bar__item fd-icon-tab-bar__item--informative">
+      <a role="tab" class="fd-icon-tab-bar__tab" href="#" id="tab15">
+        <div class="fd-icon-tab-bar__container">
+          <span class="fd-icon-tab-bar__icon">
+            <i class="sap-icon--cart" role="presentation"></i>
+          </span>
+          <span class="fd-icon-tab-bar__badge fd-icon-tab-bar__badge--animated"></span>
+          <span class="fd-icon-tab-bar__counter">10</span>
+        </div>
+      </a>
+    </li>
+  </ul>
+</div>
+`;
+badge.storyName = 'Icon Tab Bar with Badge';
+badge.parameters = {
+    docs: {
+        storyDescription: `The Tab attention badge is applied as a visual eye-catcher to indicate a change within a Tab. <br>
+        The badge is a span element with <code>fd-icon-tab-bar\\_\\_badge</code> class. It can be animated by adding an additional <code> fd-icon-tab-bar\\_\\_badge--animated</code> modifier class. In this case the badge appears for a few miliseconds and disappears.`
```

Comment:
Rewrite: In this case the badge appears for a few milliseconds and then disappears.`